docs/cases/api/actions-api/cases-actions-api-intro.asciidoc

```diff
@@ -8,7 +8,7 @@ You can push {es-sec} cases to these third-party systems:
 * {jira} (including Jira Service Desk)
 * {ibm-r}
 
-NOTE: The {kib} Console supports only Elasticsearch APIs. Console doesn't allow interactions with {kib} APIs. You must use `curl` or another HTTP tool instead. For more information, refer to https://www.elastic.co/guide/en/kibana/current/console-kibana.html[Console].
+NOTE: Console supports only Elasticsearch APIs. Console doesn't allow interactions with {kib} APIs. You must use `curl` or another HTTP tool instead. For more information, refer to https://www.elastic.co/guide/en/kibana/current/console-kibana.html[Console].
```

Comment:
The target page has a new title now, and the URL can be revised to use the `{kibana-ref}` attribute, as we all discussed in team chat. You'll also need to update all the other files with this link, but I'll just mark up this instance and leave the rest to you.
```suggestion
NOTE: Console supports only Elasticsearch APIs. Console doesn't allow interactions with {kib} APIs. You must use `curl` or another HTTP tool instead. For more information, refer to {kibana-ref}/console-kibana.html[Run {es} API requests].
```